apps/widget-configurator/src/app/configurator/hooks/useWidgetParamsAndSettings.ts

```diff
@@ -42,7 +42,7 @@ export function useWidgetParams(configuratorState: ConfiguratorState, isDappMode
       height: '640px',
       chainId,
       tokenLists: tokenLists.filter((list) => list.enabled).map((list) => list.url),
-      env: getEnv(),
+      baseUrl: { environment: getEnv() },
```

Comment:
Is this a breaking change for people already using the widget?

To avoid the breaking change - and a weird type IMO - what about we keep `env` and add `baseUrl`.

`baseUrl` then takes precedence over `env`, if present.

Reply:
The widget has already breaking changes, like `partnerFee`, and I would rather not add the `env` to a library. It feels weird we have it in the first place. 

As a user of the library, I'm not sure i would know what to add there (they are not aware of our environments). Also, once I know, I don't think I would have a need to add there `pr` for example. I would find more intuitive to add any URL I want.

If something, I would be more inclined to kill the `env`  and leave only `baseUrl`

Reply:
Do we need to expose the `environment` option at all then?

Reply:
I would be ok killing it , will do so